packages/subgraph/src/packages/token/token-voting.ts

```diff
@@ -21,7 +26,14 @@ import {
 
 import {RATIO_BASE, VOTER_OPTIONS, VOTING_MODES} from '../../utils/constants';
 import {fetchERC20} from '../../utils/tokens/erc20';
+import {bigIntToBytes32} from '../../utils/bytes';
 
+function generateProposalId(plugin: Address, pluginProposalId: BigInt): string {
```

Comment:
well, you already got this. why not using it in other files ? and put it in the utils somewhere.

better name: `getProposalId`.

Reply:
outdated

Reply:
but amended